p/themes/Origine/origine.css

```diff
@@ -47,10 +47,11 @@ option {
 	padding: 0 .5em;
 }
 
-input:focus, select:focus, textarea:focus {
+input:focus, select:focus, textarea:focus, input[type="password"]:focus + a.toggle-password {
```

Comment:
This style might have to be updated for `<button>` ?

Reply:
thanks. Fixed it.